docs/src/main/paradox/kubernetes-lease.md

```diff
@@ -113,11 +113,11 @@ spec:
 
 ```
 
-To get the current leases the fully qualified name must be uses as Kubernetes has since added a Lease type.
+@@@ note
 
-```
-kubectl get leases.akka.io
-```
```

Comment:
What was exactly the issue? 
That calling `kubectl get leases.akka.io` would only return values during a split?

Reply:
correct